File: algoritmos/bubble.js
arr = [5, 2, 9, 7, 5, 48, 54, 12, 48, 15, 4, 8, 1, 415, 1, 8, 41, 54]

function bubbleSort(arr) {
    for (let i = 0; i < arr.length - 1; i++) {
        for (let j = 0; j < arr.length - i - 1; j++) {
            if(arr[j+1] < arr[j]) {
                let buffer = arr[j+1]
                arr[j+1] = arr[j]
                arr[j] = buffer
            }
        }
    }

    return arr
}

console.log(bubbleSort(arr))

File: algoritmos/bubble.py
array = [1,2,8,5,8,5,786,54,78,87,45,24,8,54,5,84,5,45,4,15,6,4,8,6,4,8]

def bubbleSort(arr:list)->list:
    n = len(arr)
    
    for i in range(n-1):
        for j in range(n-1-i):
            if arr[j] > arr[j+1]:
                arr[j+1], arr[j] = arr[j], arr[j+1]
    return arr

print(bubbleSort(array))